docs/reference/analysis/tokenfilters/asciifolding-tokenfilter.asciidoc

```diff
@@ -1,10 +1,83 @@
 [[analysis-asciifolding-tokenfilter]]
-=== ASCII Folding Token Filter
+=== ASCII folding token filter
+++++
+<titleabbrev>ASCII folding</titleabbrev>
+++++
 
-A token filter of type `asciifolding` that converts alphabetic, numeric,
-and symbolic Unicode characters which are not in the first 127 ASCII
-characters (the "Basic Latin" Unicode block) into their ASCII
-equivalents, if one exists.  Example:
+Converts alphabetic, numeric, and symbolic Unicode characters which are not in
+the first 127 ASCII characters (the "Basic Latin" Unicode block) into their
+ASCII equivalents, if one exists. For example, the filter changes `à` to `a`.
+
+This filter uses Lucene's
+https://lucene.apache.org/core/{lucene_version_path}/analyzers-common/org/apache/lucene/analysis/miscellaneous/ASCIIFoldingFilter.html[ASCIIFoldingFilter].
+
+[[analysis-asciifolding-tokenfilter-analyze-ex]]
+==== Example
+
+The following <<indices-analyze,analyze API>> request demonstrates how the
+ASCII folding token filter works.
+
```

Comment:
My preference is to say what the example *does*, rather than explain that it's an example. Something like:

> The following <<indices-analyze,analyze API>> request uses the `asciifolding` filter to drop the diacritical marks in `açaí à la carte`: